### DISCUSSION TITLE
Resize cover image

### DISCUSSION THREAD

**Discussion by AntoniosBarotsis** [adityatelange/hugo-PaperMod#610] [Q&A] [answered]
Is there some way you can resize the cover image? I would like it to take less space in the homescreen. I tried a few stuff but only managed to downscale it, thanks!

**Comment by pixelfabs** [comment, marked as answer]
Yes you can. Currently, there's 2 documented solution:

- https://github.com/adityatelange/hugo-PaperMod/pull/514
- https://github.com/adityatelange/hugo-PaperMod/discussions/475

The solution below is more of a hack. 

If you like to keep the same white box, but much smaller covers, you need to implement the list pages as grids (instead of the current 1 column). A quick and easy way is adding:

```
<div style="display:grid;grid-template-columns: repeat(auto-fit, minmax(285px, 3fr)); column-gap:0.5em;">
```

to list.html before `{{- $pages := union .RegularPages .Sections }}`

and `</div>` before `{{- if gt $paginator.TotalPages 1 }}`
